source/utilities.tex

```diff
@@ -5129,41 +5129,68 @@
 
 \begin{itemdescr}
 \pnum
-Let $n$ be \tcode{sizeof...(Variants)}. Let \tcode{m} be a pack of $n$
-values of type \tcode{size_t}. Such a pack is called valid if $0 \leq
-\tcode{m}_i < \tcode{variant_size_v<remove_reference_t<Variants}_i\tcode{>>}$ for
-all $0 \leq i < n$. For each valid pack $\tcode{m}$, let $e(\tcode{m})$
-denote the expression:
+Let \exposid{as-variant} denote the exposition-only function templates
+
```

Comment:
I would lose these newlines around the codeblock. (But I can do that during merging.)